chapter09/ard/ex3.pl

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use v5.12;
+
+use File::Slurp;
+use Regexp::Assemble;
+
+my @lines = read_file( 'patterns' );
+my $ra = Regexp::Assemble->new();
```

Comment:
Probably not what the book would have wanted, but the `Regexp::Assemble` constructor can slurp a file of patterns: `Regexp::Assemble->new( file => 'patterns' )`, though it does lose the individual warnings so fair enough.